### DISCUSSION TITLE
How to access the value of a TextField’s prompt?

### DISCUSSION THREAD

**Discussion by grantneufeld** [nalexn/ViewInspector#258] [Q&A] [answered]
Is there a way to access the `prompt` value when inspecting a `TextField`?

Such as with code like the following:

```swift
struct MyView: View {
    @Binding var myText: String = ""
    var body: some View {
        TextField("My Label", text: $myText, prompt: "My Prompt")
    }
}
```

**Comment by nalexn** [comment]
Nope, but that should be straightforward to add. Could you convert that in a ticket instead please?

**Reply by grantneufeld** [reply]
Done. Thanks! Issue #259

**Comment by nalexn** [comment, marked as answer]
Closing in favor of the issue #259
